### DISCUSSION TITLE
0.69.6-3 release is now available

### DISCUSSION THREAD

**Discussion by douglowder** [react-native-tvos/react-native-tvos#460] [Announcements]
Changes since 0.69.6-2:

* fix: improve TVFocusGuide stability on Android by @yusufyildirim in https://github.com/react-native-tvos/react-native-tvos/pull/447
* feat: add auto focus management feature to TVFocusGuide by @yusufyildirim in https://github.com/react-native-tvos/react-native-tvos/pull/450
* chore: rename setAutoFocus public method to prevent naming collisions by @yusufyildirim in https://github.com/react-native-tvos/react-native-tvos/pull/455
* Fix flow errors by @douglowder in https://github.com/react-native-tvos/react-native-tvos/pull/456
* Fix: android crash connectAnimatedNodes by @wouterds in https://github.com/react-native-tvos/react-native-tvos/pull/457
* Bump activesupport from 6.1.7 to 6.1.7.2 by @dependabot in https://github.com/react-native-tvos/react-native-tvos/pull/454

Complete change list: https://github.com/react-native-tvos/react-native-tvos/compare/v0.69.6-2...v0.69.6-3

<hr /><em>This discussion was created from the release <a href='https://github.com/react-native-tvos/react-native-tvos/releases/tag/v0.69.6-3'>0.69.6-3</a>.</em>

**Comment by coucoseth** [comment]
Just a suggestion here for you to add to the library sothat when an app is bieng uploaded to playstore it targets only android tv devices

https://developer.android.com/training/tv/start/start#leanback-req
